Source/WebCore/dom/Node.cpp

```diff
@@ -341,6 +341,15 @@ void Node::trackForDebugging()
 #endif
 }
 
+inline void NodeRareData::operator delete(NodeRareData* nodeRareData, std::destroying_delete_t)
```

Comment:
Sweet!

Reply:
We finally have a way to construct a class, which can invoke destructors based on `m_type` :)